packages/react-components/react-button/src/components/ToggleButton/useToggleButtonStyles.ts

```diff
@@ -55,6 +55,7 @@ export const useCheckedStyles = makeStyles({
   // Appearance variations
   outline: {
     backgroundColor: tokens.colorTransparentBackgroundSelected,
+    ...shorthands.borderWidth(tokens.strokeWidthThicker),
```

Comment:
just tested, and unfortunately the focus styles completely override this, so when toggling w/ the keyboard there's no visual change